src/wallet/rpcdump.cpp

```diff
@@ -572,7 +576,9 @@ UniValue dumpwallet(const JSONRPCRequest& request)
     EnsureWalletIsUnlocked();
 
     ofstream file;
-    file.open(request.params[0].get_str().c_str());
+    boost::filesystem::path filepath = request.params[0].get_str();
+    filepath = absolute(filepath);
```

Comment:
Also is there a "namespaced" name for this function? E.g., boost::filesystem::absolute? It'd be good to use the fully qualified name, otherwise it's difficult to hunt down such dependencies/lookup documentation.

I don't have a firm opinion to add to the above conversation though, except that I would add a way to get dumpwallet to serialize results across network (although dangerous if unencrypted/authenticated link) in case the user wants the wallet dump locally (RPC may even be called on a different machine let alone different directory as @aideca points out).